concepticondata/conceptlists/Kornai-2018-1400.tsv

```diff
@@ -416,7 +416,7 @@ Kornai-2018-1400-414	414	2555	duty	N			vám	portorium	cło
 Kornai-2018-1400-415	415	1693	each	G	3416	EVERY (EACH)	mind	quisque	kaźdy	all
 Kornai-2018-1400-416	416	1716	eager	A			mohó	avidus	chétny	HAS desire, lack(wait)
 Kornai-2018-1400-417	417	1296	earn	V	733	EARN	keres	mereo	zarabiać	after(=AGT HAS =PAT[money]), before(=AGT[work])
-Kornai-2018-1400-418	418	815	earth	N			föld	terra	ziemia	planet, IN/2758 space/2509, life ON, ocean ON, island ON, HAS atmosphere
+Kornai-2018-1400-418	418	815	earth	N	965	WORLD	föld	terra	ziemia	planet, IN/2758 space/2509, life ON, ocean ON, island ON, HAS atmosphere
```

Comment:
I'd unmap. WORLD is already mapped to another item.

Reply:
föld can also mean soil. Guess "terra" too, but not sure about that one, not a native Latin speaker lol